### DISCUSSION TITLE
Issue with Firedrake installation

### DISCUSSION THREAD

**Discussion by sarshia** [firedrakeproject/firedrake#3389] [Firedrake support]
Hello,

I am trying to install Firedrake on a new laptop after having installed it on my old laptop. However, I am getting the following error:

Running firedrake-install
Installing command line tools...
Checking brew doctor...
Installing required packages via homebrew...
Traceback (most recent call last):
  File "/Users/arshiasinghal/Desktop/Rice/Research/firedrake-install", line 1651, in <module>
    for p in json.loads(check_output(["brew", "info", "--installed", "--json"])))
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/arshiasinghal/Desktop/Rice/Research/firedrake-install", line 677, in check_output
    return subprocess.check_output(args, stderr=subprocess.STDOUT, env=os.environ).decode()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['brew', 'info', '--installed', '--json']' returned non-zero exit status 1.

Install log saved in /Users/arshiasinghal/Desktop/Rice/Research/firedrake-install.log

Here is the [firedrake-install.log](https://github.com/firedrakeproject/firedrake/files/14171499/firedrake-install.log). Does this look like an issue related to an incompatibility between the old and new computers or perhaps a missing dependency?

**Comment by connorjward** [comment]
You look to be having issues with homebrew. Can you make sure that it is up-to-date? I think you want to run `brew upgrade`. You want the command `brew doctor` to not raise any issues.
I also notice that you are using the system Python rather than a homebrew-installed Python. You should run `$(brew --prefix)/bin/python3 firedrake-install` to ensure that the correct Python is used.
